docs/Using-Fleet/Automations.md

```diff
@@ -68,21 +68,17 @@ Follow the steps below to configure Jira or Zendesk as a ticket destination:
 6. Select **Enable vulnerability automations** and choose **Ticket**.
 7. Under **Ticket destination**, select your ticket destination and select **Save**.
 
-The Jira and Zendesk ticket destinations are currently in beta.
+The Jira and Zendesk ticket automations are currently in beta.
 
 ## Policy automations
 
-Policy automations are triggered if a policy is newly failing on at
-least one host.
+Policy automations are triggered if a policy is newly failing on at least one host. Policy automations are triggered separately for each newly failing policy. 
 
-> Note that a policy is "newly failing" if a host updated its response from "no response" to "failing"
-> or from "passing" to "failing."
+> Note that a policy is "newly failing" if a host updated its response from "no response" to "failing" or from "passing" to "failing."
```

Comment:
```suggestion
> Note that a policy is "recently failing" if a host updated its response from "no response" to "failing" or from "passing" to "failing."
```

Reply:
I like your instinct here. The "newly failing" wording felt awkward to me too at first. But I'm still leaning toward keeping it because I think it draws a sharper line around what counts as a triggering event.

Reply:
Sounds good to me.